app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -385,8 +391,19 @@ private static void addVideoCodecDetails(Context context, LinearLayout layout, M
     private static void addMediaDetails(Context context, MediaStream stream, LinearLayout layout) {
 
         if (stream != null) {
-            if (stream.getCodec() != null && stream.getCodec().trim().length() > 0) {
-                String codec = stream.getCodec().equals("dca") || stream.getCodec().equals("DCA") ? "DTS" : stream.getCodec().equals("ac3") || stream.getCodec().equals("AC3") ? "Dolby" : stream.getCodec().toUpperCase();
+            if (stream.getProfile() != null && stream.getProfile().contains("Dolby Atmos")) {
+                addBlockText(context, layout, "ATMOS");
+                addSpacer(context, layout, " ");
+            } else if (stream.getProfile() != null && stream.getProfile().contains("DTS:X")) {
+                addBlockText(context, layout, "DTS:X");
+                addSpacer(context, layout, " ");
+            } else if (stream.getCodec() != null && stream.getCodec().trim().length() > 0) {
+                String codec =
+                    stream.getProfile() != null && stream.getProfile().contains("DTS-HD") ? "DTS-HD" :
+                    stream.getCodec().equals("dca") || stream.getCodec().equals("DCA") ? "DTS" :
+                    stream.getCodec().equals("eac3") || stream.getCodec().equals("EAC3") ? "DD+" :
+                    stream.getCodec().equals("ac3") || stream.getCodec().equals("AC3") ? "DD" :
+                    stream.getCodec().toUpperCase();
```

Comment:
This should be replaced with a switch statement or continuing the if-else statements